TcpSocket.Example/Program.cs

```diff
@@ -34,8 +34,8 @@ private static async Task Main(string[] Args)
             var Logger = LoggerFactory.Create(Builder =>
             {
                 Builder.SetMinimumLevel(LogLevel.Trace);
-                Builder.AddConsole();
-                Builder.AddDebug();
+                //Builder.AddConsole();
+                //Builder.AddDebug();
```

Comment:
Can remove comments if need be. Both AddConsole and AddDebug methods are not supported in .NET 4.8.